### PR TITLE
Added Goal Tracker to Main Page

### PR DESCRIPTION
**Pull request by yellow-tulips**

### CODE REVIEW

**Review by joerossi215 — APPROVED** [review]
Looks good!!

**Review by farzanaisrat — APPROVED** [review]
Looks good

**Review by tasnhubaislam — APPROVED** [review]
Looks Great!

File: index.html [hunk 1]
```diff
@@ -50,8 +50,36 @@ <h2><span class = "auto-type"> </span></h2>
        
         </div>
     </div>
+
+
+    <div class = "goalTracker">
+        <div class = "tracker">
+            <h2>Goal Tracker</h2>
+            <p>The Goal Tracker helps you keep track of your mental health goals by letting you add or delete a goal.
```
Comment (Taraque-A) [inline index.html:58]: It's look pretty good.

File: index.html [hunk 2]
```diff
@@ -50,8 +50,36 @@ <h2><span class = "auto-type"> </span></h2>
...
 
+
+
+
+    <script src="main.js"></script>
+
+
     <footer>
 
         <p><small>All content copyright &copy; 2023, HelpingHand</small></p>
```
Comment (Taraque-A) [inline index.html:85]: This part is really nice.

File: index.html [hunk 3]
```diff
@@ -50,8 +50,36 @@ <h2><span class = "auto-type"> </span></h2>
...
+                Remember, you don't need to do everything all at once! 
+            </p>
+        
+            <form>
+            <input type="text" id="goal" placeholder="Enter a Goal ... "/>
+            </form>
+          <span onclick="addGoal()" class="addBtn">&#43</span>
+        </div>
+        
+        <ul id="goalList">
```
Comment (Taraque-A) [inline index.html:69]: This look really nice.

**Pull request merged**